### PR TITLE
switch voices and languages

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

File: examples/foundational/15a-switch-languages.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+logger.remove(0)
+logger.add(sys.stderr, level="DEBUG")
+
+current_language = "English"
+
+
+async def switch_language(llm, args):
+    global current_language
+    current_language = args["language"]
+    return {"voice": f"You are answers from now on should be in {current_language}."}
```
Comment (kwindla) [inline examples/foundational/15a-switch-languages.py:47]: This should be "Your answers ..." rather than "You are answers ..."

**Comment by kwindla** [comment]
I think the transcription debug log from the Daily transport is very helpful when building/testing/debugging:

https://github.com/pipecat-ai/pipecat/blob/c25f6e56e7b01aa11b300cef79b9eeb156512553/src/pipecat/transports/services/daily.py#L853

Could we add a similar log line for the Whisper STT service?

**Review by kwindla — APPROVED** [review]
lgtm

**Comment by aconchillo** [comment]
Thanks! Addressed both comments.

**Pull request merged**